.github/workflows/test.yml

```diff
@@ -24,3 +24,19 @@ jobs:
       fail-fast: false
     with:
       os: ${{ matrix.os }}
+  xNuts:
+    needs: linux-unit-tests
+    uses: salesforcecli/github-workflows/.github/workflows/externalNut.yml@main
+    strategy:
+      fail-fast: false
+      matrix:
+        os: ['ubuntu-latest', 'windows-latest']
+    with:
+      packageName: '@salesforce/packaging'
+      externalProjectGitUrl: 'https://github.com/salesforcecli/plugin-packaging'
+      command: 'yarn test:nuts:package'
+      os: ${{ matrix.os }}
+      preSwapCommands: 'yarn upgrade @salesforce/core; npx yarn-deduplicate; yarn install'
+      preExternalBuildCommands: 'npm why @salesforce/core --json'
```

Comment:
do we need this, or was it for debugging?

Reply:
Arguably both.  I added it for debugging but it won't harm anything to keep it.  That way when tsc fails when building the xNUT repo we can see why, at least for `@salesforce/core`.  I can remove it though.